pkg/validator/messages/messages.go

```diff
@@ -96,4 +96,8 @@ const (
 	HostNetworkFailure = "Host network should not be configured"
 	// HostNetworkSuccess message
 	HostNetworkSuccess = "Host network is not configured"
+	// ImageRegistryFailure message
+	ImageRegistryFailure = "The image registry is not allowed or must match the allowed registries list"
```

Comment:
Let's break out separate messages for whitelist and blacklist, since they have separate IDs

Reply:
👍 Updated so that blacklist and whitelist both have failure and success messages.